From ff9cf3bbae500caf1231cb87b7e5255217066b08 Mon Sep 17 00:00:00 2001
From: Mete Alpaslan <mete@pardus.org.tr>
Date: Wed, 1 Dec 2010 21:20:08 +0200
Subject: [PATCH 1/4] yali.gui.partition_gui:Set filesystem type as default automatically

If creating physical volume is selected, filesystem type should be
setted automatically.

BUG:FIXED:15340
---
 yali/gui/partition_gui.py |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/yali/gui/partition_gui.py b/yali/gui/partition_gui.py
index 36083b4..5bbea20 100644
--- a/yali/gui/partition_gui.py
+++ b/yali/gui/partition_gui.py
@@ -217,8 +217,6 @@ class PartitionWidget(QtGui.QWidget, Ui_PartitionWidget):
             storageGuiHelpers.fillFilesystemMenu(self.filesystemMenu, self.origrequest.format,
                                                  availables=restricts)
             QObject.connect(self.filesystemMenu, SIGNAL("currentIndexChanged(int)"), self.formatTypeChanged)
-            default_format = self.filesystemMenu.findText("ext4")
-            self.filesystemMenu.setCurrentIndex(default_format)
             self.resizeRadio.hide()
             self.resizeSlider.hide()
             self.resizeSpin.hide()
-- 
1.7.2.2

